src/app/shared/utils/course-mapper.ts

```diff
@@ -0,0 +1,14 @@
+import { Course, CourseFromService } from '../models/Courses';
+
+export function transformCourseFromService(
+  coursesFromService: CourseFromService[]
+): Course[] {
+  return coursesFromService.map(courseFromService => ({
+    id: courseFromService.id,
+    name: courseFromService.name,
+    code: courseFromService.code,
+    description: courseFromService.description,
+    sessions: courseFromService.sessions,
+    duration: courseFromService.sessionDuration,
```

Comment:
here we could unstructure the variable coursesFromService?

Reply:
Done!